### ISSUE TITLE
Problem for generating 3D voxel grids

### ISSUE THREAD

**Issue by yangziyi1990** [gersteinlab/ThermoNet#5]
I installed HTMD with 2.0.8 version, when I run the command to generate tensors (parameterized 3D voxel grids):
`python gends.py -i p53_variants.txt -o p53_direct_stacked_16_1 -p ./ --boxsize 16 --voxelsize 1`
the problem happened:
<img width="876" alt="image" src="https://user-images.githubusercontent.com/20346636/177535247-0f14ba70-a160-4409-94a4-a47b7dce0747.png">
Is this due to the high version of HTDM?

**Comment by hdieckhaus** [comment]
@yangziyi1990 can I ask how you solved this problem? I am having the same issue with htmd version 2.0.6

**Comment by hdieckhaus** [comment]
In case anyone else runs into this problem in the future, this is the solution I arrived at after several hours of debugging:

The issue appears to be that htmd wants me to register before using its API, but the registration function is apparently broken, so the first time ThermoNet imports htmd, it breaks.

To work around this (I have HTMD version 2.0.5), I did the following:
In the **ThermoNet.utils.pdb_utils.py** file:

1. Comment out this line:
`
from htmd.ui import *
`
2. Replace it with this line:
`
from moleculekit.molecule import Molecule
`

After this change, the gends.py script should now run without error and also without prompting you to register. Hope this saves someone some time in the future!
